crates/swc_ecma_minifier/src/compress/optimize/if_return.rs

```diff
@@ -200,6 +200,9 @@ where
                 match s {
                     Stmt::Decl(Decl::Var(v)) => {
                         if v.decls.iter().all(|v| v.init.is_none()) {
+                            if last_idx == 0 {
+                                break;
+                            }
```

Comment:
I'm not exactly sure if this is the correct fix, but it stops a panic and the output seems ok so 🤷‍♂️

Reply:
Yeah, it's correct fix, thanks!